src/uca-camera.c

```diff
@@ -926,6 +926,7 @@ uca_camera_is_recording (UcaCamera *camera)
     gboolean result;
     g_return_val_if_fail (UCA_IS_CAMERA (camera), FALSE);
     g_object_get (camera, "is-recording", &result, NULL);
+    camera->priv->is_recording = result;
```

Comment:
Will I go to hell?

Reply:
Does it mean you like it or I _will_? 😄 Anyway, it's not good yet, more on Monday...

Reply:
Got rid of this.